va-gov/includes/logo-350px-base64.html

```diff
@@ -0,0 +1 @@
cIUlyxBsidhjjY4N0F1/w0BKFbTizXdKeGBlRGSEPy68wIJFHxSSCm6DKcqflGYK+sQdR/HmtZQcKCD/1Qot/Z454hLoXd7oRp5EoMdq0nCI5meNfv6oYal7i6rgPFGFyouGo7VrH3UwpxElCJpyFOPlVPekV4h6VOYdwysffYHCVz9Aov5rFFcHodlJFWCWc35vP0y9Ihp+vqSUSSVnHavG3xYUY+7ckbQ5oUPl14scma+Ha2ZuMWttgY7Ha8u3zWLjyag4JDwR+bWHdgj8V0OAg709/EAIyO7AmUxboBTRChRLTnxWcT/w+38imVucasx5F3ZZuUv7xZXDEDN9cJ21w8VRhkq6cVqy0ownnivG9N/pUFRiw2frbYgJp43pCgdWfpCGg0cbyJp1oo5ug2srTuCrrX5wxKchNDUVsYlx8DeHQBtogg+pYV+2RE+esIMHZCIVoeWBmbrnta21lU4p61B34jhdyedh7/oDQMN6/Hl6PwRZQhFO0bbVn1dg5dpmdOlkwKRxwYilS/e4WD9KXhzF8LFxmDLBF+d39EPLgVz7J8uq9DO3hyE8wPpYRdnWhzyA+RfD8p8Af/sn7RD4F0GgHfH+OMAKvLwU2an3Py6XH5XajXwn9r4waXk2/jHZqO05/rpyW8+eWl14tzHa3dnFGDe1kDnq0SNNA2q7YiS38oMHWpCcGEgZ4DzExPhiWHo0Skrq8fZbR2FAM3IoqWVIjEGz0QwNkacpwox6OkIN9aN3A1K+/nrK5lLmt6S8FuamKhyneckWFOM3oyhRcVECyit16JoagPBQPbZ/XUVWQh227RIugQbznorBFxuOY/nqeowfbsZDDwxEdEA+jqz+wPrVu72MM7MCqD3vuBPHt73gAUU70v1RY6I98a8dAu2shh/fg4Jw/01IVypXptgOR8p3VN87KvHd1/eg2xfl5q6dauhQXXPIFt/JqZt+Qz9079KIhe+X0XUQ/ZVdHqJUcoODdPjwYzePtUtnXywgayI2IRTTb+wOq78PFTKCMXaoEfEBx+EqP4JhaRW0JpIDc/NRtOXvw8BuzUgKqkfGlgbYaULtqis6Y+KEZHzwcQWuvzoGh47QV1uHQB6etWLZJ5S4iNNTZd6OxZ8W0GOwDz79eDSmXRUAZ8EaZ/Z7Zfb3l6Uan6v2aUiNc/y2qmjbWx44tiNd6eb28D8FgXbE+6vobjfy/fLoNpqrK/ogxDfKuWSncWhtta+ug3+9LcpSr4lLrNNcNnog0tJ8MWP2AQT563HwSDO+3G7DozMSUUhJhnmvn8BVk0PoqNKX2m81dGNFG4A9Q/H1vmaYQkJwGanZuiYDLOEWXDq8I32hBaCKwl076RdVCP3e3Q0YQbsRM+YUYtjgYCXZ4BTlR1oQXPJpGapqatHUasHS93+D++8KRpyhFI2HSm0bPqvSv7s2Qrekxrl1aIxtzM5jO8imEemF/Hae7q9i/LVX8ueGQDvi/bkh+i/Lr8w5RRQseADV2Fiy6YK0+Ix1+7QX1h70jWypr9WEBjXZ0pJDNJ3T6jQ9usbB1xSJNxYVsDYOdEv1wd4D9bhyMjXMrA7yXgORndtAIzV6Glo30OxkNqIjjLiwbzCp12bcN6sMIy8JQJ8eJiJX8ovX1iCZ1OzGzDZcOtiPbIsgGmdvwp599di06QTWrqjCgPSOeOyRAbj5Wn+cn+JPy6tHHXvXLHWtuS9YP2OfBdkGPOW6O/OqaUtLaVlL2CeZZ7Nj8C+DXnvG7RD4T4KAbJnbw68LAtyap/OXYZ81Lcl31kdRD1Jo9p7pg13GAR1OIOXiGltMTLpOH+mv3bm/HlVNkZj/yh7s2W/FpRebsG5LI/46IxKFJW3o2d2MYHp/uuamHNxzWxQRahi27axWXoatVhft/TqVltmd9x3nYZkGBfQBJ5Tv2OEByCexeuCYC0/8tTd6dTPSviu9FEdEuSqLNjqy1nylObiiu25uWQjd2zq+7BnuuHvfoa3b3GAWSle09NpDOwT+dyHQjnh/tX3vPnST6o/p2bf7yn16Sgdof3NtugaDo+hdYkC1LT6xhzYgKkRbUlmv0QWlk72wA0tW1FJqwQfr14uhWxssvr6oaW1C7y5BGDQwAPX1bWQv6NDa2KIULz7fLDhSDPL703h5BGLD7UhKCsP4sZ1gb9qPyKBol5/e4So6tsJRuMGq27ElTru4Jgi5tbZsWm59EnVbX+fHDMq8o7AWyLNuD+0Q+N+GQDvi/XX3PylfZSRcKR10Txtw0YEjtOIPzaS+HXS+v+tdi/CYaiT3DLXHJXWFy5SnratJ0WiDDZrde8thie6DrCM5qKrTwUDHuuszazDowkjs35uPuA4JSIwGyktqMHrMebA25SA6LAYWcz1slYUuXbPJWVl30FV2qEJbuNFPu7ooDvsb6Ram1rEPfvj7jcOsb7z26dfNbvC2U7m/7mHWXvufGwLtiPfnhugvkp9CbEJJKmoyfcCAtIxMXMVnukPRdZnW24bOumLEDzMhydfpNEZbHUERETBH9tXafAq09rZQjc6nGY1U7TWFRqO2ageMun4IDGlDXdkhevzt5bTpcxx1JcdRW6TT5B6v11dscWLTrkisgRHwsVP1jb7uIlzvzBpSv0qsrLnBoKhyIZm9Inju6Pa/7RD4H4dAO+L9rxoACgEL9asQ3bRp6b6fbGwZXJ2nvQwwDKad766Twhw+9HmHXoPaYKZbIGO/YMToW1xWl15jDKY1s9JmtFlM9IrYgIZaHzQE+8C+owJHy2Nw3GLA8f0abKUkMCwoobHfHTx9W5sWidVHjmzN/waUpy8E38S337VDoB0CAoF2xPvfNw7Yp2JSsYLXbw6xXB/R4v4TF3beddjQCy3OXsTNaez+Trx2TITLj0rFLj1NqJMhgX2EiRhUdzNjmU2glnJhzmOMzkKC9uBFprbdnfw0Bxdt364cJ3pA6D30O0l5//eBtr1F7RD4eSDw/3UsXyDY7OcnAAAAAElFTkSuQmCC
```

Comment:
Do we need to base64 encode the image? Or could we include the svg markup inline instead?

Reply:
I think we can include the SVG inline; I was just following GitHub's 503 as a template. Do you know if we have VA.gov's logo as an SVG already?

Reply:
Nvm, found it here - https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/design/vadotgov/assets/VA%20logo/SVG

Reply:
Ah, there's a formatting error in the `va-logo-color.svg` file so it can't be opened...trying to find a new one.

Reply:
Moving this to its own ticket so we can proceed today - https://github.com/department-of-veterans-affairs/vets.gov-team/issues/14260